Gameplay.js

```diff
@@ -3,8 +3,8 @@
 * @description Manages the boards and user interaction during gameplay (ship placement and attacking)
 * @member rows The number of rows the boards have
 * @member cols The number of columns the boards have
-* @member numShips The number of ships each player will place
-* @member turn Which player's turn it is (players are false and true, aka 0 and 1)
+* @member numShips The number of ships each player has.
+* @member turn Which player's turn it is (players are false and true, when it is the LHS player's turn, turn=0, when it's the RHS player's turn, turn=1).
```

Comment:
This isn't actually true because numbers and booleans are different things in JavaScript